build/storage/tests/it/ptf_tests/system_tools/const.py

```diff
@@ -3,3 +3,8 @@
 #
 
 STORAGE_DIR_PATH = "ipdk/build/storage"
+SHARE_DIR_PATH = "shared"
+NQN = "nqn.2016-06.io.spdk:cnode0"
+SPDK_PORT = 5260
+NVME_PORT = "4420"
```

Comment:
It's a string here because create_and_expose_subsystem_over_tcp and create_virtio_blk within disk_infrastructure accept a string. We can change it here to an integer but then functions within disk_infrastructure need to be changed.

Reply:
changed